### ISSUE TITLE
[LATER] Add .deb as installation method

### ISSUE THREAD

**Issue by MagicLike** [openandroidinstaller-dev/openandroidinstaller#132]
For users who flash a lot (and just for convenience), it would be nice to have DEB packages (and also maybe a repo in the future, to ship updates directly to the user.)

**Comment by tsterbak** [comment]
Is there an added benefit in having a .deb compared to flatpak?

**Comment by MagicLike** [comment]
> Is there an added benefit in having a .deb compared to flatpak?

Ohhh there is a Flatpak in the works? :eyes:

**Comment by tsterbak** [comment]
Yeah, the code is already on main and work to add it to flathub is in progress.

If you want to test it, you can run
`flatpak-builder build-dir org.openandroidinstaller.OpenAndroidInstaller.json`
`flatpak-builder --user --install --force-clean build-dir openandroidinstaller.OpenAndroidInstaller.json`
`flatpak run openandroidinstaller.OpenAndroidInstaller.json`
(Following this tutorial: https://docs.flatpak.org/en/latest/first-build.html)

This will build the flatpak and install it on your user account. You will even get a nice start menu entry for it :D

**Comment by MagicLike** [comment]
> Yeah, the code is already on main and work to add it to flathub is in progress.
> 
> If you want to test it, you can run `flatpak-builder build-dir org.openandroidinstaller.OpenAndroidInstaller.json` `flatpak-builder --user --install --force-clean build-dir openandroidinstaller.OpenAndroidInstaller.json` `flatpak run openandroidinstaller.OpenAndroidInstaller.json` (Following this tutorial: https://docs.flatpak.org/en/latest/first-build.html)
> 
> This will build the flatpak and install it on your user account. You will even get a nice start menu entry for it :D

Gonna try that out in the next few days! Also to answer your question: There is no real benefit, besides people that are used to it / want to make it the traditional way / who like using apt...

**Comment by tsterbak** [comment]
You might not even need to try that. In a few hours there might be the official flatpak release online :)

I keep the .deb in mind.

**Comment by MagicLike** [comment]
> You might not even need to try that. In a few hours there might be the official flatpak release online :)
> 
> I keep the .deb in mind.

Nice! - maybe you should also keep .rpm in mind...

**Comment by MagicLike** [comment]
Another nice thing would be to host repos for the Debian / Fedora based system or add OAI to existing ones...
